roles/ipareplica/library/ipareplica_prepare.py

```diff
@@ -601,10 +602,20 @@ def main():
     ansible_log.debug("-- CA_CRT --")
 
     cafile = paths.IPA_CA_CRT
-    if not os.path.isfile(cafile):
-        ansible_module.fail_json(
-            msg="CA cert file is not available! Please reinstall"
-            "the client and try again.")
+    if install_ca_cert is not None:
+        if not os.path.isfile(cafile):
+            ansible_module.fail_json(
+                msg="CA cert file is not available! Please reinstall"
+                "the client and try again.")
+    else:
+        if is_ipa_client_configured(on_master=True):
```

Comment:
`is_ipa_client_configured` could be None according to `ansible_ipa_replica.py`. When is `is_ipa_client_configured` defined in `ipalib.facts`. There should be a check to make sure that it is not None before it is used this way.